keps/prod-readiness/sig-storage/3294.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3294
+alpha:
+  approver: "@wojtek-t"
```

Comment:
I'm sorry, I won't get to it - can you please reassign to @johnbelamaric ?

Reply:
Thank you for letting me know. I've updated the KEP to reassign to him.
/assign @johnbelamaric

Please review this KEP. @johnbelamaric